sdk/filter/filter.go

```diff
@@ -1,15 +1,18 @@
 package filter // import "github.com/hypertrace/goagent/sdk/filter"
 
-import "github.com/hypertrace/goagent/sdk"
+import (
+	"github.com/hypertrace/goagent/sdk"
+	"github.com/hypertrace/goagent/sdk/filterutils"
```

Comment:
Do we really need a `filterutils` package of can this just in the same filter package?

Reply:
https://github.com/hypertrace/goagent/pull/183#discussion_r939807930

Reply:
Aah I see. Ok. Let's use `"github.com/hypertrace/goagent/sdk/filter/result"` then instead of `"github.com/hypertrace/goagent/sdk/filterutils"`? I don't like utils since it ends up being a dumping ground of "other" functionality.